Packs/CortexXDR/Integrations/CortexXDRIR/test_data/get_incidents_list_dedup.json

```diff
@@ -70,52 +70,6 @@
     },
     {
         "incident_id": "4",
-        "creation_time": 100000001,
-        "modification_time": 1575813875168,
```

Comment:
Is it really necessary to revert here?

Reply:
it will make it easier for me to unrevert